src/main/java/edu/cornell/kfs/concur/businessobjects/ConcurEventNotificationProcessingResultsDTO.java

```diff
@@ -25,6 +25,11 @@ public ConcurEventNotificationProcessingResultsDTO(ConcurEventNoticationVersion2
         this(eventType, processingResults, reportNumber, travelerName, travelerEmail, new ArrayList<String>());
     }
     
+    public ConcurEventNotificationProcessingResultsDTO(ConcurEventNotificationProcessingResultsDTO dtoToCopy) {
```

Comment:
this may be another example of a class that should be renamed to reflect it just isn't used with event notification

Reply:
I'll add this class to the scope of the user story for renaming the event notification V2 web service.